buildSrc/src/main/groovy/openhouse.maven-publish.gradle

```diff
@@ -56,6 +56,9 @@ publishing {
 
         configurePom(pom)
       }
+      mavenJava(MavenPublication) {
```

Comment:
Having both will fail repository upload, Jfrog doesnt allow multiple publications with same coordinates

Reply:
Yea, it worked fine in mavenLocal so thought will give it a try. Reverted